sdk/tests/performance/webgl_webcodecs_video_frame/index.html

```diff
@@ -0,0 +1,173 @@
+<!-- This was created from the demo of webodecs-blogpost-demo.glitch.me, and added the webgl support for test only purpose. -->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="origin-trial"
+    content="AoSY6Q4OOuuZVXTqOwJlfk4n77EL0esumtLRHj+9V97JoFfLq4AKsWlza8A8HmxTx1PU7SSzpjWK6F62bwWLPQYAAAB3eyJvcmlnaW4iOiJodHRwczovL3dlYmNvZGVjcy1ibG9ncG9zdC1kZW1vLmdsaXRjaC5tZTo0NDMiLCJmZWF0dXJlIjoiV2ViQ29kZWNzIiwiZXhwaXJ5IjoxNjA1NDc0OTQ4LCJpc1N1YmRvbWFpbiI6dHJ1ZX0=">
+  <title>WebCodecs API demo: Encoding and Decoding</title>
+  <style>
+    canvas {
+      padding: 10px;
+      background: gold;
+    }
+
+    button {
+      background-color: #555555;
+      border: none;
+      color: white;
+      padding: 15px 32px;
+      width: 150px;
+      text-align: center;
+      display: block;
+      font-size: 16px;
+    }
+  </style>
+  <script src="webgl_webcodecs_video_frame.js"></script>
+</head>
+
+<body>
+  <canvas id="src" width="640" height="480"></canvas>
+  <button onclick="playPause()">Pause</button>
+  <canvas id="dst" width="640" height="480"></canvas>
+  <script>
+    let codec_string = "vp8";
+    let keep_going = true;
+
+    function playPause() {
+      keep_going = !keep_going;
+      let btn = document.querySelector("button");
+      if (keep_going) {
+        btn.innerText = "Pause";
+      } else {
+        btn.innerText = "Play";
+      }
+    }
+
+    async function startDrawing() {
+      let cnv = document.getElementById("src");
+      var ctx = cnv.getContext('2d', { alpha: false });
+
+      ctx.fillStyle = "white";
+      let width = cnv.width;
+      let height = cnv.height;
+      let cx = width / 2;
+      let cy = height / 2;
+      let r = Math.min(width, height) / 5;
+      let drawOneFrame = function (time) {
+        let angle = Math.PI * 2 * (time / 5000);
+        let scale = 1 + 0.3 * Math.sin(Math.PI * 2 * (time / 7000));
+        ctx.save();
+        ctx.fillRect(0, 0, width, height);
+
+        ctx.translate(cx, cy);
+        ctx.rotate(angle);
+        ctx.scale(scale, scale);
+
+        ctx.font = '30px Verdana';
+        ctx.fillStyle = 'black';
+        const text = "😊📹📷Hello WebCodecs 🎥🎞️😊";
```

Comment:
Would you consider linking to http://crbug.com/1142754 and/or writing up the extension formally as a WebGL extension proposal, and linking to that?

Reply:
I tried, but the linking to proposal was too long to be displayed appropriately.